ibm-cloud-pak-for-data/scripts/bastionPrep.sh

```diff
@@ -14,27 +14,27 @@ runuser -l $SUDOUSER -c "chmod 600 /home/$SUDOUSER/.ssh/id_rsa*"
 
 # Remove RHUI
 
-rm -f /etc/yum.repos.d/rh-cloud.repo
+#rm -f /etc/yum.repos.d/rh-cloud.repo
```

Comment:
if the commented code isn't needed, just remove it from the file

Reply:
@bmoore-msft thanks. this is addressed.